library/core/src/iter/traits/iterator.rs

```diff
@@ -2851,7 +2851,7 @@ pub trait Iterator {
         Product::product(self)
     }
 
-    /// Lexicographically compares the elements of this [`Iterator`] with those
+    /// [Lexicographically](../cmp/trait.Ord.html#lexicographical-comparison) compares the elements of this [`Iterator`] with those
```

Comment:
Can you also change the rest of these to intra-doc links? There are about 15 links you added, I'd prefer all of them to be changed.